<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyOptionsWidget</class>
 <widget class="QWidget" name="ProxyOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>284</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rdbAutoProxy">
        <property name="text">
         <string>Automatic connection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbExplorerProxy">
        <property name="text">
         <string>Import proxy settings from Internet Explorer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbFireFoxProxy">
        <property name="text">
         <string>Import proxy settings from Mozilla Firefox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbManualProxy">
        <property name="text">
         <string>Specify the proxy manually</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbManualProxySettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lneProxyHost">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lneProxyPort">
          <property name="text">
           <string notr="true">1080</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbProxyPort">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>62</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>62</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>1080</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chbProxyUserPassword">
        <property name="text">
         <string>Name and password is required to connect</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lneProxyUser">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lneProxyPassword">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>0</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lneProxyHost</tabstop>
  <tabstop>spbProxyPort</tabstop>
  <tabstop>chbProxyUserPassword</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chbProxyUserPassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>lneProxyUser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbProxyUserPassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>lneProxyPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdbManualProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbManualProxySettings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
